fhir-search/src/main/java/com/ibm/fhir/search/reference/ReferenceParameterHandler.java

```diff
@@ -1,88 +0,0 @@
-/*
- * (C) Copyright IBM Corp. 2020
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-package com.ibm.fhir.search.reference;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.stream.Collectors;
-
-import com.ibm.fhir.config.FHIRRequestContext;
-import com.ibm.fhir.model.resource.SearchParameter;
-import com.ibm.fhir.search.parameters.QueryParameterValue;
-import com.ibm.fhir.search.reference.value.CanonicalUrlHandlerImpl;
-import com.ibm.fhir.search.reference.value.IdHandlerImpl;
-import com.ibm.fhir.search.reference.value.ParameterValueHandler;
-import com.ibm.fhir.search.reference.value.TypeIdHandlerImpl;
-import com.ibm.fhir.search.reference.value.UrlHandlerImpl;
-
-/**
- * Reference Parameter Handler controls the access to the underlying implementation of the handler
- *
- * <li><b>reference</b></li>
- * <li>[parameter]=[url]</li>
- * <li>[parameter]=[url|version] - canonical url</li>
- * <li>[parameter]=[type]/[id]</li>
- * <li>[parameter]=[id]</li>
- */
-public class ReferenceParameterHandler {
-
-    //@formatter:off
-    private static final List<ParameterValueHandler> handlers = Arrays.asList(
-            new CanonicalUrlHandlerImpl(),
-            new IdHandlerImpl(),
-            new TypeIdHandlerImpl(),
-            new UrlHandlerImpl()
-        );
```

Comment:
Do you also want to delete ParameterValueHandler, CanonicalUrlHandlerImpl, IdHandlerImpl, TypeIdHandlerImpl, and UrlHandlerImpl since they're only used by this class?

Reply:
Oh, good catch. I thought they were inner classes. As standalone I'll definitely get rid.

Reply:
Removed